src/TDockWidget.h

```diff
@@ -0,0 +1,46 @@
+#ifndef MUDLET_TDOCKWIDGET_H
+#define MUDLET_TDOCKWIDGET_H
+/***************************************************************************
+ *                                                                         *
```

Comment:
You really do want to add your name + contact EMail to the header for each file in the year that you work on them.

How are you going to get universal praise and admiration for your deft coding and cunning algorithms if no-one knows it was you? :grinning:

{And hiding your details really won't stop the tar-and-feathers brigade if you do a really monumental cock-up - and as someone who occasionally does make mistakes, I feel strongly that feathers keep you warm and I am sure there is _some_ use for tar!} :rofl: